setting-up-appsmith/aws-ami.md

```diff
@@ -0,0 +1,185 @@
+- Click the “Add Rule” button in the "Inbound Rule" section and add new rules for HTTP, HTTPS, and SSH access using the following guidelines:
+  - Type: Use the pre-defined types “HTTP”, “HTTPS”, and “SSH”.
+  - Source: Use “Anywhere” to allow access from anywhere, or use “Custom IP” and specify an IP address range.
+  ![Inbound Rules](../.gitbook/assets/aws-inbound-rules.png)
+- Click the “Create” button to save your changes.
+
+# Step 4: Deploy Appsmith On An AWS Cloud Server
+```
+At the end of this step, your Appsmith stack will be running on an AWS cloud server.
+```
+
+The next step is to launch a cloud server with the Appsmith Amazon Machine Image (AMI) running on it. The AWS Console lets you do this in just a couple of clicks. Follow these steps:
+
+- From the Amazon EC2 dashboard, select the “AMIs” option in the “Images” menu.  
+  ![AMI](../.gitbook/assets/aws-AMI.png)
+- Search for the Appsmith Stack by entering the search term `appsmith` in the search bar at the top.
+  ![Search AMI](../.gitbook/assets/aws-search-ami.png)
+
+- Select the image in the list of search results and click the “Launch” button.
+
+- On the resulting detail page, review the available server sizes. Select the server size you wish to use and click “Review and Launch” to proceed.
+  ![Instance Preview](../.gitbook/assets/aws-preview.png)
+
+- On the review page, click the “Edit security groups” link
+
+- On the “Configure Security Group” page, choose the option to “Select an existing security group”. Find the security group you created in Step 3 and select it. Click the “Review and Launch” button to proceed.
+
+- Verify that the correct key pair (created in Step 2) will be used for the server.
+
+- Confirm your selection by hitting the “Launch Instances” button.
+
+The AWS Console will now begin spinning up the new server.
+![Launch](../.gitbook/assets/aws-launch.png)
+
+The process usually takes a few minutes. Use the EC2 Dashboard to check the status of the server. Once the server has launched, you will be able to obtain its public IP address from the EC2 Dashboard, as shown below: 
+
+![EC2 Detail](../.gitbook/assets/aws-ec2-detail.png)
+
+At this point, you should be able to browse to the cloud server, by entering the cloud server IP address or DNS name directly into your browser’s address bar. You should now see your web app home page as shown below:
+![Login Page](../.gitbook/assets/aws-login-page.png)
+
+
+## Step 5: Login & start using Appsmith
+```
+At the end of this step, you will have logged in to Appsmith and build your internal tools
+```
+
+To log in to the Appsmith dashboard, follow these steps:
+- You will need to register an account to log in
+  - Go to the Sign Up page, fill in your user's email & password, then click Sign Up
+  ![Appsmith Signup Page](../.gitbook/assets/aws-appsmith_signup.png)
+
+Once you completed the registration, you will be moved to the Personal Organization page. Now you can begin to create your custom app. 
```

Comment:
On sign-up, you'll be redirected to your Personal Organization page.
Create an app within this organization or create a new organization for your app
Start building

PS - Ideally this should link to the section in the document that tells users about org / app etc. Since we don't have that yet, let's go ahead with this as of now.